packages/product/src/services/product-variant.ts

```diff
@@ -23,6 +24,30 @@ export default class ProductVariantService<
     this.productService_ = productService
   }
 
+  async retrieve(
+    productVariantId: string,
```

Comment:
comment: In the core, we throw on undefined entity ids in `retrieve`-methods. Do we want to add that either here or in the "gateway" service? I am fine to leave it out, as I think it's reasonable to assume data is passed correctly, but maybe it would make for a better DX.